packages/react-components/src/molecules/TabbedContent.tsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
out of curiosity - why do we no longer need these? are non-null assertions now allowed?